src/components/Answer.jsx

```diff
@@ -0,0 +1,36 @@
+export function Ans({ correct, text, click, picked, check }) {
+  return check ? (
+    <button
+      disabled={check}
+      className={`border   ${
+        picked && correct
+          ? "bg-[#94D7A2]"
+          : picked && !correct
+          ? "bg-[#F8BCBC]"
+          : "border-[#4d5b9e7e] text-[#29326477]"
+      } px-5 py-1 rounded-[8px] font-medium text-xs`}
```

Comment:
I have seen you are using classname package [here](https://github.com/izzett222/react-quizzical/pull/1/files#diff-307073a6118b6ce3181b31970b407b3ac08d0709d336180bac417bd310f55799) and I think also here it could make this logic more understandable.